src/Plugin/FieldEncryptionProvider/CoreStrings.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\field_encrypt\Plugin\FieldEncryptionProvider\CoreStrings.
+ */
+
+namespace Drupal\field_encrypt\Plugin\FieldEncryptionProvider;
+
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\encrypt\EncryptionProfileManagerInterface;
+use Drupal\encrypt\EncryptServiceInterface;
+use Drupal\field_encrypt\Plugin\FieldEncryptionProvider\EncryptProviderBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a Field Encryption Provider for core fields with string values.
+ *
+ * @FieldEncryptionProvider(
+ *   id = "core_strings",
+ *   label = @Translation("Core string fields"),
+ *   field_types = {
```

Comment:
This precisely demonstrates the properties one would select for potential encryption values within the UI.

One would select "value",  "summary", "uri", or "title" from the list of properties, as I outlined before. We can filter out common attributes, like "id" that would cause breakage in the system.